cmdstanpy/stanfit.py

```diff
@@ -1330,3 +1337,133 @@ def save_csvfiles(self, dir: str = None) -> None:
         :param dir: directory path
         """
         self.runset.save_csvfiles(dir)
+
+
+def from_csv(
+    dir: str = None, method: str = None
```

Comment:
I would just ask for a list of paths for csv files. Using a directory is error-prone when there are multiple different runs in the same folder.

Reply:
how about:
- required argument `dir`
- optional arguments either `pattern` - the glob or `names` - a list of csv filenames.
if only `dir` is specified, then get all csvfiles in the directory.

Reply:
I think the simplest api is something like

`def from_csv(path: str | list[str], method: str = None)`

If path is a list, it must be a list of individual files. If it is not a list, it would be a glob if it contains a '*' character, or a dir otherwise. This would generate a little more boilerplate but avoids the issue of having mutually exclusive arguments

Reply:
Also, I think we should recommend users trying pathlib

    paths = list(pathlib.Path("path/to/folder").glob("my_run_x_*.csc"))

And then manually check that the files are correct. Otherwise debugging why the files are not seen could be hard.